### PR TITLE
Replacing Apply Page placeholder Image with Lottie

### PR DESCRIPTION
**Pull request by ss1106**
Replacing ApplyPage FAQ placeholder image with a lottie.

### CODE REVIEW

File: src/utility/LottieOptions.tsx [hunk 1]
```diff
@@ -6,13 +6,22 @@ import projectManagementLottie from "../images/lottie/projectManagement_lottie.j
...
+    preserveAspectRatio: "xMidYMid slice",
+  },
+};
 export const projectTeamLottieOptions = {
   loop: true,
   autoplay: true,
   animationData: projectTeamLottie,
   rendererSettings: {
-    preserveAspectRatio: "xMidYMid slice",
+    preserveAspectRatio: "xMidYMin slice",
```
Comment (Niyousha99) [inline src/utility/LottieOptions.tsx:24]: Why did this change?

File: src/pages/ApplyPage.tsx [hunk 2]
```diff
@@ -99,6 +102,9 @@ function ApplyPage() {
           data-aos="fade-right"
           data-aos-duration="1000"
         />
+        <S.LottieDiv>
+          <Lottie options={applyPageTeamLottieOptions} />
+        </S.LottieDiv>
```
Comment (Niyousha99) [inline src/pages/ApplyPage.tsx:107]: From here you can add a `LottieDiv` component in `ApplyPage.styles.ts` to add css rules for the position and size.

File: src/utility/LottieOptions.tsx (hunk 1 above)
Comment (ss1106) [reply src/utility/LottieOptions.tsx:24]: I was testing the preserveAspectRatio to see its effect, so I deleted and then added the line back, but that's the original line, I reverted all the changes so nothing should change I believe

**Review by Niyousha99 — APPROVED** [review]
Follow up issue to improve scaling of Lottie: #341 

Otherwise looks good!

**Review by TakahiroFujitaSJB — APPROVED** [review]
Code and Animation looks good to me! (Check with size 300 and above)

**Pull request merged**